guides/01_getting-started/02_key-features.md

```diff
@@ -39,7 +39,7 @@ There are three arguments in the `Interface` constructor to specify where this c
 * `description`: which accepts text, markdown or HTML and places it right under the title.
 * `article`: which also accepts text, markdown or HTML and places it below the interface.
 
-![annotated](/assets/guides/annotated.png)
+![annotated](https://github.com/gradio-app/gradio/blob/main/guides/assets/annotated.png?raw=true)
```

Comment:
If we put these assets in the public directory, i think they will get served by the App, this will be much more efficient than serving them from github.

Reply:
understood, but since this would require changing the path inside the guide md files, it would break assets in the current website before we migrate. So will do this after the migration is done